src/main/java/fr/quatrevieux/araknemu/game/fight/turn/action/move/validators/TackleValidator.java

```diff
@@ -0,0 +1,88 @@
+package fr.quatrevieux.araknemu.game.fight.turn.action.move.validators;
+
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+
+import org.apache.commons.lang3.RandomUtils;
+
+import fr.arakne.utils.maps.constant.Direction;
+import fr.arakne.utils.maps.path.Path;
+import fr.arakne.utils.maps.path.PathStep;
+import fr.quatrevieux.araknemu.data.constant.Characteristic;
+import fr.quatrevieux.araknemu.game.exploration.interaction.action.move.validator.PathValidationException;
+import fr.quatrevieux.araknemu.game.fight.fighter.Fighter;
+import fr.quatrevieux.araknemu.game.fight.fighter.PassiveFighter;
+import fr.quatrevieux.araknemu.game.fight.map.BattlefieldMap;
+import fr.quatrevieux.araknemu.game.fight.map.FightCell;
+import fr.quatrevieux.araknemu.game.fight.turn.action.move.Move;
+import fr.quatrevieux.araknemu.game.fight.turn.action.move.MoveFailed;
+
+final public class TackleValidator implements PathValidatorFight {
+    @Override
+    public Path<FightCell> validate(Move move, Path<FightCell> path) throws PathValidationException {
```

Comment:
Tackle in dofus retro do not works like that. It should use 2 validators :
- The first will check the tackle value **only for the start cell** and return a MoveFailed result if fail
- The second will stop the character on the first enemy adjacent cell, and return the MoveSuccess result with truncated path